### PR TITLE
Add configure orientation for devcard

### PR DESCRIPTION
**Pull request by iamncdai**
Add configure orientation for devcard
- `default`: Vertical (Default)
- `wide`: Horizontal

Example usage:
```yaml
jobs:
  devcard:
    runs-on: ubuntu-latest
    permissions:
      contents: write
    steps:
      - name: devcard
        uses: dailydotdev/action-devcard@3.0.0
        with:
          user_id: ${{ secrets.USER_ID }}
          type: wide
```

Hope my contribution will be accepted. Thank you for the great Action!

### CODE REVIEW

File: src/index.ts
Comment (omBratteng) [inline src/index.ts:22]: Please do a validation check that the type is either `default` or `wide`.

**Review by omBratteng — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please do not include the dist/index.js and the yarn.lock changes.

File: src/index.ts
Comment (iamncdai) [reply src/index.ts:22]: I will update. Thank you for comment.

Comment (kopancek) [reply src/index.ts:22]: Maybe even use an enum type for the `type` parameter. Or string union type. E.g. something like:

```typescript
// enum
enum DevCardType {
  Vertical = 'default',
  Horizontal = 'wide',
  Twitter = 'x',
}

// union
type DevCardType = 'default' | 'wide' | 'x';
```

You can then use the `DevCardType` for the validation as well.

Comment (iamncdai) [reply src/index.ts]: Hi @omBratteng, I just updated the code. Only change the `src/index.ts` and `action.yml` files, as well as check the validity of the `type`.

File: src/index.ts
Comment (kopancek) [inline src/index.ts:39]: Thank you for the effort and sorry for another comment. 

I think a cleaner solution would be to define `DevCardType` as enum, because you can then do
```typescript
Object.values(DevCardType).includes(type)
```

So whenever the enum changes, this check would automatically work as well without the need to change the code.

Comment (iamncdai) [reply src/index.ts:39]: Thank you. I will update soon.

Comment (iamncdai) [reply src/index.ts]: Hi @kopancek, I've updated.

**Review by kopancek — APPROVED** [review]
Nice, looks great! 👍 :shipit:

**Comment by iamncdai** [comment]
> Nice, looks great! 👍 :shipit:

@kopancek Sorry, I just discovered the error. Please wait for me to update a bit.

**Review by kopancek — CHANGES_REQUESTED** [review]
Sorry, tests are failing now, because of the default value. Also might make sense to fix the README.md

File: action.yml
Comment (iamncdai) [reply action.yml]: Hi @kopancek
Great, I've updated. Sorry for the serious confusion.

File: README.md
Comment (iamncdai) [reply README.md]: Hi @kopancek,
Great, I've updated. Sorry for the serious confusion.

**Review by omBratteng — COMMENTED** [review]
lgtm, will merge once @kopancek approves

**Review by omBratteng — APPROVED** [review]
lgtm, will merge once @kopancek approves

**Review by kopancek — APPROVED** [review]

**Pull request merged**
